### DISCUSSION TITLE
更新到6.1.0的时候网站翻译出现问题

### DISCUSSION THREAD

**Discussion by happytbs** [bs-community/blessing-skin-server#424] [Q&A]
### Blessing Skin 版本

6.1.0

### PHP 版本

8.0

### 出现问题时所使用的浏览器

Chrome, Microsoft Edge

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [x] 是

### 出现了什么问题？

![image](https://user-images.githubusercontent.com/74001578/175023781-59506fa6-d8a6-4c86-aa8e-367f9d82f99c.png)
如图所示，侧边栏全部变成英文了，但是右边的设置有的又是中文。（机子在国外）

### 重现步骤

出错前我先点击了系统里面的更新，然后等待更新完毕后我点了确定，接着就出错了。

**Comment by happytbs** [comment]
注意：我清理了缓存

还有其他人测试也是这样

**Comment by mochaaP** [comment]
`php artisan optimize:clear`
无法复现

**Comment by happytbs** [comment]
![image](https://user-images.githubusercontent.com/74001578/175435903-974da407-4190-4b29-90ca-5433bf3579c8.png)
特别奇怪，就算我设置中文简体也还是这样
